src/player/player.rs

```diff
@@ -51,13 +51,13 @@ pub trait SpotifyPlayerDelegate {
     fn notify_playback_state(&self, position: u32);
 }
 
-#[derive(Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
I think (I need to double check) that the derived PartialEq impl won't check for equality on the nested String here, so there would be Alsa("foo") == Alsa("bar")

Reply:
Double checked on [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2eb03c1532fce3300c3aa1e7a32f6d01). `derive` handles it well I suppose :)